gcp/modules/couchdb/main.tf

```diff
@@ -17,6 +18,7 @@ variable "limits_memory" {}
 
 # Secret variables
 variable "secret_couchdb_admin_username" {}
+
```

Comment:
same as before, why is needed a new line here?

Reply:
This is what `terraform fmt` does. `terraform fmt` is a standard for terraform formatting and I would say we should go with it, otherwise, it becomes a debate on personal preferences of spaces vs. tabs :D.

I was wrong when I mentioned I think I know why is this happening at the standup, it's not because of alphabetical ordering, but rather because of the comment before the variable. If the variable block is more than one line (and TF apparently counts this with comments), it will be separated by a new line.

I did a little test:
```
variable "a" {}
variable "b" {}
variable "c" {}

variable "f" {}
variable "e" {}
variable "d" {}

# Variable a with a comment
variable "g" {}
# Variable b with a comment
variable "h" {}
# Variable c with a comment
variable "k" {}

variable "i" {
  default = "abc"
}
variable "j" {
  default = "abc"
}
variable "k" {
  default = "abc"
}
```

is reformatted into:
```
variable "a" {}
variable "b" {}
variable "c" {}

variable "f" {}
variable "e" {}
variable "d" {}

# Variable a with a comment
variable "g" {}

# Variable b with a comment
variable "h" {}

# Variable c with a comment
variable "k" {}

variable "i" {
  default = "abc"
}

variable "j" {
  default = "abc"
}

variable "k" {
  default = "abc"
}

```

Reply:
I am also not a big fan of this Terraform's formatting rule, but we can live with that.